subprojects/docs/src/docs/userguide/authoring-builds/software-product/composite_builds.adoc

```diff
@@ -80,6 +78,23 @@ include::sample[dir="samples/build-organization/composite-builds/basic/kotlin",f
 
 More details about tasks that depend on included build tasks are below.
 
+[[included_plugin_builds]]
+=== Including builds that define Gradle plugins
+
+A special case of included builds are builds that define Gradle plugins.
+These builds should be included using the `includeBuild` statement inside the `pluginManagement {}` block of the settings file.
+Using this mechanism, the included build may also contribute a settings plugin that can be applied in the settings file itself.
+
+.Including a plugin build
+====
+include::sample[dir="snippets/developingPlugins/testingPlugins/groovy/include-plugin-build",files="settings.gradle[tags=include-build]"]
+include::sample[dir="snippets/developingPlugins/testingPlugins/kotlin/include-plugin-build",files="settings.gradle.kts[tags=include-build]"]
+====
+
+NOTE: Including plugin builds via the plugin management block is an incubating feature.
+You may also use the normal `includeBuild` mechanism to include plugin builds.
```

Comment:
Maybe instead of saying "normal `includeBuild` mechanism", rephrase it to something like "stable `includeBuild` mechanism outside of `pluginManagement` block".

Reply:
Good point. I rephrased this.